cmd/factory.go

```diff
@@ -36,7 +37,16 @@ func (f *factory) Create(name string) (cmd Cmd, err error) {
 }
 
 func (f *factory) SetConfig(config davconf.Config) {
-	httpClient := boshhttpclient.CreateDefaultClient(nil)
+	var httpClient boshhttpclient.Client
+	var caCert *x509.CertPool
+
+	if len(config.CaCert) != 0 {
+		caCert = x509.NewCertPool()
+		caCert.AppendCertsFromPEM([]byte(config.CaCert))
```

Comment:
let's use https://github.com/cloudfoundry/bosh-utils/blob/master/crypto/x509.go#L11 to catch bad certs